package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "linter-pep8",
   "main": "./lib/main",
-  "version": "1.3.2",
-  "description": "Linter plugin for pep8",
+  "version": "1.4.0",
```

Comment:
Also, since the settings will be changing this needs to be a major release as it can potentially break an existing user's setup.